src/client/conn.rs

```diff
@@ -63,7 +63,10 @@ use tower_service::Service;
 
 use super::dispatch;
 use crate::body::HttpBody;
-use crate::common::{task, exec::{BoxSendFuture, Exec}, Future, Pin, Poll};
+use crate::common::{
```

Comment:
Seems unrelated to the change?

Reply:
I accidentally format it while debugging I guess.